### PR TITLE
[docs] ADD Arm Compile Dockerfile

### PR DESCRIPTION
**Pull request by FreeOnePlus**
Add the Dockerfile file of the Docker compiled image under the ARM architecture

# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docker/arm/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+# install dependencies
+RUN yum install -y byacc patch automake libtool perf vim make which file ncurses-devel gettext-devel unzip bzip2 zip util-linux \
+    ccache wget git python2 bison java-11-openjdk-devel java-1.8.0-openjdk-devel
+
+# clean cache
+RUN yum clean all
+
+# install maven 3.6.3
+RUN mkdir -p /usr/share/maven /usr/share/maven/ref && \
+    wget -q -O /tmp/apache-maven.tar.gz https://doris-thirdparty-repo.bj.bcebos.com/thirdparty/apache-maven-3.6.3-bin.tar.gz && \
```
Comment (hf200012) [inline docker/arm/Dockerfile:32]: Use the official download link directly

File: docker/arm/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    tar -xzf /tmp/apache-maven.tar.gz -C /usr/share/maven --strip-components=1 && \
+    rm -f /tmp/apache-maven.tar.gz && \
+    ln -s /usr/share/maven/bin/mvn /usr/bin/mvn
+
+# install nodejs
+RUN wget https://doris-thirdparty-repo.bj.bcebos.com/thirdparty/node-v16.3.0-linux-arm64.tar.xz \
```
Comment (hf200012) [inline docker/arm/Dockerfile:38]: Use the official download link directly

File: docker/arm/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+RUN alternatives --set java java-11-openjdk.aarch64 && alternatives --set javac java-11-openjdk.aarch64
+
+
+# squash all layers to reduce the image size
+FROM scratch
+COPY --from=builder / /
+ENV JAVA_HOME="/usr/lib/jvm/java-11" \
+    MAVEN_HOME="/usr/share/maven" \
+    REPOSITORY_URL="https://doris-thirdparty-repo.bj.bcebos.com/thirdparty" \
```
Comment (hf200012) [inline docker/arm/Dockerfile:77]: Leave empty here, do not specify third-party resource libraries by default

Comment (FreeOnePlus) [reply docker/arm/Dockerfile]: all modified

File: docker/arm/Dockerfile (hunk 2 above)
Comment (FreeOnePlus) [reply docker/arm/Dockerfile]: OK

File: docker/arm/Dockerfile (hunk 1 above)
Comment (FreeOnePlus) [reply docker/arm/Dockerfile]: ok~

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**